lesson-4/src-solution/styles/Spinner.css

```diff
@@ -0,0 +1,41 @@
+.spinner {
+  margin: 100px auto 0;
+  width: 70px;
+  text-align: center;
+}
+
+.spinner > div {
+  width: 18px;
+  height: 18px;
+  background-color: #333;
+
+  border-radius: 100%;
+  display: inline-block;
+  -webkit-animation: sk-bouncedelay 1.4s infinite ease-in-out both;
```

Comment:
what is this plz? create-react-app will autoprefix raw css files

Reply:
This code in Spinner.css is a copy paste, from referenced lib.

I'll make it more explicit